ideas/307-breaking-whiisper.md

```diff
@@ -0,0 +1,131 @@
+---
+id: 307-breaking-whisper
+title: Breaking Whisper
+status: Draft
+created: 2018-09-24
+category: swarm / SITG experiment
+lead-contributor: corpetty
+contributors:
+exit-criteria: no
+success-metrics: yes
+clear-roles: yes
+future-iterations: yes
+---
+
+## Preamble
+    Idea: 307-breaking-whisper
+    Title: SITG Experiment #2: Breaking Whisper
+    Status: Draft
+    Created: 2018-09-24
+
+## Summary
+1) Understanding the limits of whisper (with and without optimizations)
+
+## Swarm Participants
```

Comment:
Are all these confirmed?

Reply:
No they are not confirmed, I will reach out to each to start filling in more requirements from the sitg hackmd doc.